src/modules/roc_sndio/ibackend.h

```diff
@@ -55,6 +57,9 @@ class IBackend {
                                  const char* driver,
                                  const char* input,
                                  const Config& config) = 0;
+
+    //! Append supported dirvers to Array
+    virtual void get_drivers(core::Array<DriverInfo>& arr) = 0;
```

Comment:
@gavv, could we use `core::List` instead of `core::Array` ?

Reply:
We don't want it. It will require to allocate every DriverInfo object on heap.

Reply:
Well, we about allocations during the array grow?

Reply:
With List, there is always one allocation per object. With Array, the allocations can be amortized so we will have log(N) allocations per N objects.